extensions/vertx-graphql/runtime/pom.xml

```diff
@@ -12,6 +12,7 @@
 
     <artifactId>quarkus-vertx-graphql</artifactId>
     <name>Quarkus - Quarkus - Vert.x GraphQL - Runtime</name>
+    <description>Query REST services using GraphQL</description>
```

Comment:
@rsvoboda isn't GraphQL a different approach than REST?

Reply:
yeah, `Query the API using GraphQL` would be better I guess, I will create followup PR

Reply:
PR is here: https://github.com/quarkusio/quarkus/pull/9814

Thanks for spotting this